### PR TITLE
Prepost

### PR DESCRIPTION
**Pull request by bpramod**
Changing permissions validation
Config files names are changed
Plugin Scripts timeout is increased

### CODE REVIEW

File: VMBackup/main/PluginHost.py [hunk 1]
```diff
@@ -77,12 +77,13 @@ def pre_check(self):
 
         if not os.path.isfile(self.configLocation):
             self.logger.log('Plugin host Config file does not exist in the location ' + self.configLocation, True, 'Error')
-            self.configLocation = './main/PluginHost.conf'
+            self.configLocation = './main/VMSnapshotPluginHost.conf'
         
+        permissions = self.get_permissions(self.configLocation)
```
Comment (vityagi) [inline VMBackup/main/PluginHost.py]: will this throw exception if file does not exist ?

File: VMBackup/main/PluginHost.py [hunk 2]
```diff
@@ -77,12 +77,13 @@ def pre_check(self):
...
         if not os.path.isfile(self.configLocation):
             self.logger.log('Plugin host Config file does not exist in the location ' + self.configLocation, True, 'Error')
             errorCode =CommonVariables.FailedPrepostPluginhostConfigNotFound
-        elif not self.validate_permissions(self.configLocation):
+        elif (int(permissions[0]) == 7) or not (int(permissions[1]) == 0 or int(permissions[1]) == 4) or not (int(permissions[2]) == 0 or int(permissions[2]) == 4):
```
Comment (vityagi) [inline VMBackup/main/PluginHost.py]: remove int(permissions[0]) == 7) as by-default this file has 744 permissions

File: VMBackup/main/ScriptRunner.py [hunk 3]
```diff
@@ -81,16 +81,25 @@ def get_config(self):
         try:
             with open(self.configLocation, 'r') as configFile:
                 configData = json.load(configFile)
```
Comment (vityagi) [inline VMBackup/main/ScriptRunner.py]: configData is under with block. is it not required to declare outside to access it outside ?

Comment (bpramod) [inline VMBackup/main/ScriptRunner.py]: In python,variables are scoped to innermost method it belongs to.if/for/with blocks doesn't count . So scoping is not issue but in our case if we don't enter the "with" block and access the configData, it throws error which is caught in the exception handling.

File: VMBackup/main/PluginHost.py (hunk 2 above)
Comment (bpramod) [inline VMBackup/main/PluginHost.py]: ok

File: VMBackup/main/PluginHost.py (hunk 1 above)
Comment (bpramod) [inline VMBackup/main/PluginHost.py]: it is handled in the get_permissions method itself

**Pull request merged**